packages/big-design/src/components/InlineMessage/InlineMessage.tsx

```diff
@@ -68,7 +72,7 @@ export const InlineMessage: React.FC<InlineMessageProps> = memo(
         {props.onClose && (
           <GridItem>
             <MessagingButton
-              iconOnly={<CloseIcon size="medium" title="Close." />}
```

Comment:
we lost the `.`?

Reply:
I think it's okay to remove it. I think the initial intention was to create a pause when a screen reader lands on the button but it should occur naturally since a SR will naturally stop on the element.